sklearn/metrics/classification.py

```diff
@@ -1674,10 +1674,13 @@ def hamming_loss(y_true, y_pred, labels=None, sample_weight=None):
     y_type, y_true, y_pred = _check_targets(y_true, y_pred)
     check_consistent_length(y_true, y_pred, sample_weight)
 
-    if labels is None:
-        labels = unique_labels(y_true, y_pred)
-    else:
-        labels = np.asarray(labels)
+    if labels is not None:
+        warnings.warn(
+                      "This parameter was deprecated in version 0.21"
+                      " and will be removed in version 0.23 as the"
+                      " number of labels is identical"
```

Comment:
I think this will confuse the user who may not realise it was only used for the number. Better to say "it was unused"

Reply:
To answer both this and unrelated changes remark: you accepted another PR with exact same "issues". And one would think that at least here people are consistent..  Great job! 👍

Reply:
You're quite right that I missed this issue on the other pr and should not have approved it! Thanks for helping me out, @sandramne. Of course we make mistakes. We require two approvals for that reason